packages/desktop-gui/src/runs/setup-project.jsx

```diff
@@ -374,7 +374,7 @@ class SetupProject extends Component {
   _setupProject () {
     if (this.state.newProject) {
       return dashboardProjectsApi.setupDashboardProject({
-        projectName: this.state.projectName,
+        projectRoot: this.props.project.path,
```

Comment:
This is the fix - instead of using `options.projectRoot`, we should just pass the `projectRoot` when we call `setupDashboardProject`.